test/common/components/SampleFeature.unit.spec.jsx

```diff
@@ -7,21 +7,17 @@ import SampleFeature from '../../../src/js/common/components/SampleFeature.jsx';
 describe('<SampleFeature>', () => {
   it('should be empty when not enabled', () => {
     const tree = SkinDeep.shallowRender(
-      <div>
-        <SampleFeature/>
-      </div>
+      <SampleFeature/>
```

Comment:
skin-deep 1.0 seems more strict about shallow rendering, and wasn't rendering `<SampleFeature/>` as it was written before.